jib-core/src/main/java/com/google/cloud/tools/jib/tar/TarStreamBuilder.java

```diff
@@ -33,17 +31,11 @@
 /** Builds a tarball archive. */
 public class TarStreamBuilder {
 
-  @FunctionalInterface
-  private interface TarArchiveOutputStreamConsumer {
-    void accept(TarArchiveOutputStream tarArchiveOutputStream) throws IOException;
-  }
-
   /**
-   * Maps from {@link TarArchiveEntry} to function that outputs the entry onto a {@link
-   * TarArchiveOutputStream}. The order of the entries is the order they belong in the tarball.
+   * Maps from {@link TarArchiveEntry} to a {@link ByteSource}. The order of the entries is the
```

Comment:
Since we already have `Blob`, we can just use that in place of `ByteSource` - it's basically the same thing except with a much smaller API (just `writeTo`). The alternative is to replace `Blob` with using `ByteSource`.